src/esss_fix_format/cli.py

```diff
@@ -291,7 +311,35 @@ def _process_file(filename, check, format_code):
     return changed, errors, formatter
 
 
-def _main(files_or_directories, check, stdin, commit, format_code):
+def run_black_on_python_files(files, check) -> Tuple[bool, bool]:
+    """
+    Runs black on the given files (checking or formatting).
+
+    Black's output is shown directly to the user, so even in events of errors it is
+    expected that the users sees the error directly.
+
+    :return: a pair (formatted, error)
+    """
+    py_files = [x for x in files if x.suffix == '.py']
+    error = False
+    formatted = False
+    if py_files:
+        if check:
+            click.secho('Checking black...', fg='cyan')
+        else:
+            click.secho('Running black...', fg='cyan')
+        args = ['black']
+        if check:
+            args.append('--check')
+        args.extend(str(x) for x in py_files)
+        status = subprocess.call(args)
+        error = not check and status != 0
+        formatted = check and status == 1
```

Comment:
Perhaps `formatted` should be called `should_format`, to be more clear? (From black: `Return code 1 means some files would be reformatted.`)

(also in line 364)

Reply:
Definitely, not sure what I was thinking...